### PR TITLE
add desktop version configuration

### PR DESCRIPTION
**Pull request by bohaz**
-Add modified html and css files for desktop version configuration.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,131 +21,180 @@
...
+  
+        <div class="profesional-art-container-Data-Dashboard">
+          <h2 class="profesional-art-h2">Data Dashboard<br>Healthcare</h2>
+          <p class="profesional-art-p">A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required. Has been the<br>industry's standard.</p>
+          <ul class="profesional-art-ul">
+            <li><img src="img/Taghtml2.png"></li>
+            <li><img src="img/Tagbootstrap2.png"></li>
+            <li><img src="img/Tagruby2.png"></li>
+          </ul> 
+        </div>
```
Comment (Shedrack-Sunday) [inline index.html:102]: - [ ] Impressive work you did here on the card 👏 . However, note that the white background with the `button` should display on over on each card, and the card should return back to the blurry version when not hoverd.
- This functiioality was given as a guide on the template, and i suggest you kindly do so for all cards involved. Good job 👍 

## Card expected hover design:
image

## When not hovered:
image

File: index.html [hunk 2]
```diff
@@ -179,6 +231,7 @@ <h2 class="about-me-h2">About me</h2>
       </ul>
       <img src="img/blackbar.png" alt="" class="black-bar">
     </div>
```
Comment (Shedrack-Sunday) [inline index.html:233]: -- [ ] Kindly add a little bit of padding space between the social icons to perfect your work and fit the design given 👍 . Good job

## Design:
image

## Figma:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @bohaz         👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of GitHub features ✔️   
- Nice UI design👍 

  
# Required Changes  🌵 

- [x] You made some impressive reviews in the comments, which I must say is a job well done. However, there's a few you need to adjust

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @bohaz     :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**